### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by williamjamir**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -9,20 +10,64 @@ Qt Style Sheet Inspector
...
-Observation
------------
+The repository demo_qt_inspector_ contains a full example of a Qt Application with an inspector widget being called by a shortcut action.
+
+.. _demo_qt_inspector: https://github.com/williamjamir/demo_qt_inspector
+
+
+See the demo in action:
+
+.. image:: https://github.com/williamjamir/demo_qt_inspector/blob/master/images/qt_inspector_demo.gif
```
Comment (fogo) [inline README.rst:25]: Great gifs, good job!

File: README.rst [hunk 2]
```diff
@@ -9,20 +10,64 @@ Qt Style Sheet Inspector
...
+        :width: 10px
+        :height: 10px
+        :scale: 10 %
+    
+
+
+Observation
+-----------
+
+    It need PyQt5 to work but it doesn't have it as a dependency.
```
Comment (fogo) [inline README.rst]: Minor typo here, `It needs`

File: README.rst [hunk 3]
```diff
@@ -9,20 +10,64 @@ Qt Style Sheet Inspector
...
-It need PyQt5 to work but it doesn't have it as a dependency, as testing with PyQt5 pypi proved
-unreliable (may be changed in the future).
 
 
 Features
 --------
+view current style sheet of application during runtime
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    The inspector only checks for style sheets that was applied to the QApplication, it's the topmost and any change here can be propagated to all child's.
```
Comment (fogo) [inline README.rst]: Some typos here:

`that was applied` -> `that were applied`
`child'`s` -> `children`

Comment (williamjamir) [reply README.rst]: Commit https://github.com/ESSS/qt_style_sheet_inspector/pull/5/commits/11115fdbf8c963366ee8c58ccca8e79c425144a8 fix this issue =)

File: README.rst (hunk 2 above)
Comment (williamjamir) [reply README.rst]: Commit https://github.com/ESSS/qt_style_sheet_inspector/pull/5/commits/11115fdbf8c963366ee8c58ccca8e79c425144a8 fix this issue ;)

**Review by nicoddemus — APPROVED** [review]

**Comment by nicoddemus** [comment]
I fixed some indentation issues myself (and indented paragraph is treated as a "quote" by RST). LGTM we should merge as soon as CI passes. 👍

**Pull request merged**